recipe/meta.yaml

```diff
@@ -76,16 +76,16 @@ extra:
 
 # Package: arrow
 # Title: R Integration to 'Apache' 'Arrow'
-# Version: 0.12.1
+# Version: 6.0.0.2
```

Comment:
I know this is all in comments but it has drifted pretty far from the actual DESCRIPTION, not sure if that matters

Reply:
Oh, sure! It was just a quick edit to make the CI rebuild.

Sorry about that!